include/bluetooth/bluetooth.h

```diff
@@ -505,9 +520,35 @@ struct bt_le_oob {
  *
  * @param id  Local identity, in most cases BT_ID_DEFAULT.
  * @param oob LE related information
+ *
+ *  @return Zero on success or error code otherwise, positive in case
+ *  of protocol error or negative (POSIX) in case of stack internal error
  */
 int bt_le_oob_get_local(u8_t id, struct bt_le_oob *oob);
 
+/**
+ * @brief Set OOB data during LE SC paring procedure
+ *
+ * This function allows to set OOB data during the LE SC paring procedure. The
+ * function should only be called in response to the oob_data_request() callback
+ * provided that LE SC method is used for pairing.
+ *
+ * The user should submit OOB data according to the information received in the
+ * callback. This may yield three different configurations: with only local OOB
+ * data present, with only remote OOB data present or with both local and
+ * remote OOB data present.
+ *
+ * @param conn Connection object
+ * @param oobd_local Local OOB data or NULL if not present
+ * @param oobd_remote Remote OOB data or NULL if not present
+ *
+ *  @return Zero on success or error code otherwise, positive in case
+ *  of protocol error or negative (POSIX) in case of stack internal error
+ */
+int bt_le_sc_oob_data_set(struct bt_conn *conn,
```

Comment:
Since `bt_le_sc_oob_data_set` operates on bt_conn (it's the first parameter), wouldn't this API be more suited for `conn.h`?

Reply:
Yes, that would make more sense.

Reply:
okay, moved it to conn.h